### PR TITLE
Text spotting 0002

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
@IRDonch

File: models/intel/index.md [hunk 1]
```diff
@@ -168,9 +168,9 @@ Deep Learning models for text spotting (simultaneous detection and recognition).
...
-| [text-spotting-0001-detector](./text-spotting-0001-detector/description/text-spotting-0001-detector.md)                                        | 185.169             | 26.497    |
-| [text-spotting-0001-recognizer-encoder](./text-spotting-0001-recognizer-encoder/description/text-spotting-0001-recognizer-encoder.md)          | 2.082                | 1.328      |
-| [text-spotting-0001-recognizer-decoder](./text-spotting-0001-recognizer-decoder/description/text-spotting-0001-recognizer-decoder.md)          | 0.002                | 0.273      |
+| [text-spotting-0002-detector](./text-spotting-0002-detector/description/text-spotting-0002-detector.md)                                        | 185.169             | 26.497    |
+| [text-spotting-0002-recognizer-encoder](./text-spotting-0002-recognizer-encoder/description/text-spotting-0002-recognizer-encoder.md)          | 2.082                | 1.328      |
+| [text-spotting-0002-recognizer-decoder](./text-spotting-0002-recognizer-decoder/description/text-spotting-0002-recognizer-decoder.md)          | 0.002                | 0.273      |
```
Comment (IRDonch) [inline models/intel/index.md]: Do all of the complexity/size numbers stay the same?

Comment (Ilya-Krylov) [reply models/intel/index.md]: there is no architectural difference between 0001 and 0002, only the weights

**Comment by vladimir-dudnik** [comment]
@Ilya-Krylov does changes from [PR-856](https://github.com/opencv/open_model_zoo/pull/856) also needed for new model?

**Comment by Ilya-Krylov** [comment]
@vladimir-dudnik thanks

**Comment by IRDonch** [comment]
You got some merge conflicts.

**Comment by Ilya-Krylov** [comment]
@IRDonch merge conflicts have been resolved

**Comment by IRDonch** [comment]
Jenkins: test this please

**Pull request merged**